opensrp-chw-referral/src/main/java/org/smartregister/chw/referral/activity/ReferralDetailsViewActivity.kt

```diff
@@ -147,7 +147,7 @@ open class ReferralDetailsViewActivity : SecuredActivity() {
             if (!StringUtils.isEmpty(memberObject!!.servicesBeforeReferralOther)) {
                 preReferralManagement.append(", " + memberObject!!.servicesBeforeReferralOther)
             }
-        } catch (e: StringIndexOutOfBoundsException) {
+        } catch (e: Exception) {
```

Comment:
Codacy will complain about making this too generic. Could you possibly catch the relevant exception here?